native/more/more.react.js

```diff
@@ -39,11 +44,28 @@ const deleteAccountOptions = { headerTitle: 'Delete account' };
 const buildInfoOptions = { headerTitle: 'Build info' };
 const devToolsOptions = { headerTitle: 'Developer tools' };
 const appearanceOptions = { headerTitle: 'Appearance' };
+const friendListOptions = ({ navigation }) => ({
+  headerTitle: 'Friend list',
+  // eslint-disable-next-line react/display-name
```

Comment:
I don't know if we want to have this ignore line here

Reply:
Yeah it's correct. It's a quirky situation where `headerRight` is actually never rendered as a `<ReactComponent>`, it's just called like a function. For that reason, setting the `displayName` property on it has no effect. `react/display-name` assumes it's a React function component because it's defined in exactly the same way. In fact, it could be used as a component. But it just isn't